dev-lua/luacheck/luacheck-0.20.0.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit toolchain-funcs
+
+DESCRIPTION="A tool for linting and static analysis of Lua code"
+HOMEPAGE="https://github.com/mpeterv/luacheck"
+SRC_URI="https://github.com/mpeterv/luacheck/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="luajit test"
+
+RDEPEND="
+	dev-lua/luafilesystem[luajit=]
+	!luajit? ( >=dev-lang/lua-5.1 )
```

Comment:
Just FYI, I don't think luakit has been tested on lua 5.2 or 5.3, and I'm pretty sure the author makes no guarantees of it working with them. Woud <dev-lang/lua-5.2:= be a better version dependency?